app/services/reporting/agency_and_sp_report.rb

```diff
@@ -67,11 +67,10 @@ def service_providers
     end
 
     def facial_match_issuers
-      @facial_match_issuers ||= Profile.where(active: true).where(
-        'verified_at <= ?',
-        report_date.end_of_day,
-      ).where(idv_level: Profile::FACIAL_MATCH_IDV_LEVELS).
-        pluck(:initiating_service_provider_issuer).uniq
+      @facial_match_issuers ||= Profile.active.verified.facial_match.
+        where('verified_at <= ?', report_date.end_of_day).
+        distinct.
+        pluck(:initiating_service_provider_issuer)
```

Comment:
```suggestion
      @facial_match_issuers ||= Reports::BaseReport.transaction_with_timeout do
        Profile.active.verified.facial_match.
          where('verified_at <= ?', report_date.end_of_day).
          distinct.
          pluck(:initiating_service_provider_issuer)
      end
```

We should probably wrap this in the report transaction helper so it runs against the replica and has a longer timeout.